qdrant-landing/content/documentation/frameworks/openlit.md

```diff
@@ -0,0 +1,53 @@
+---
+title: OpenLIT
+weight: 3100
+---
+
+# OpenLIT
+
+[OpenLIT](https://github.com/openlit/openlit) is an OpenTelemetry-native LLM Application Observability tool and includes OpenTelemetry auto-instrumention for Qdrant with just a single line of code helping you ensure your applications are monitored seamlessly, providing critical insights to improve performance, operations and reliability. 
+
+For more information on how to use OpenLIT, see the [OpenLIT docs](https://docs.openlit.io/).
```

Comment:
I recommend creating a `## Further Reading` section at the end of the page to aggregate all the links to external docs. Similar to the other documentation pages.

Reply:
I guess this was missed. I don't see it.